Src/Microsoft.Dynamic/Actions/ReflectedPropertyTracker.cs

```diff
@@ -80,11 +74,7 @@ public class ReflectedPropertyTracker : PropertyTracker {
             return _propInfo.GetIndexParameters();
         }
 
-        public PropertyInfo Property {
-            get {
-                return _propInfo;
-            }
-        }
+        public PropertyInfo Property => _propInfo;
```

Comment:
I'm not really sure what difference that makes? In SourceUnit you changed Path when other properties used it...